build/webpack.base.config.js

```diff
@@ -29,6 +29,9 @@ function getCssLoaders(config) {
         {
             loader: 'css-loader',
             options: {
+                // Required to work with css-loader@4
+                // https://github.com/vuejs/vue-style-loader/issues/46#issuecomment-670624576
+                esModule: false,
```

Comment:
Looks like this was a breaking change in v4 that didn't play nicely with vue-style-loader

https://github.com/webpack-contrib/css-loader/blob/master/CHANGELOG.md#400-2020-07-25
https://github.com/vuejs/vue-style-loader/issues/46